app/src/main/java/co/kr/mvvm_study/view/adapter/ToDoListAdapter.kt

```diff
@@ -76,4 +86,10 @@ class ToDoListAdapter() : RecyclerView.Adapter<ToDoListAdapter.ToDoViewHolder>()
     fun getItem(position: Int): ToDoModel {
         return toDoItems[position]
     }
+}
+
+@BindingAdapter("toDoDate")
+fun setToDoDate(tv: TextView, date: Long) {
+    val simpleDateFormat = SimpleDateFormat("yyyy.MM.dd HH:mm:ss", Locale.getDefault())
+    tv.text = simpleDateFormat.format(date)
```

Comment:
이부분은 아까도 얘기했지만 오브젝트에 빼서 관리하는게 더 좋을듯 근데 나 밥먹고와서 이부분 한개만 더 지적할게 ㄱㄷ

Reply:
```suggestion
@BindingAdapter("toDoDate")
fun TextView.setToDoDate(date: Long) {
    val simpleDateFormat = SimpleDateFormat("yyyy.MM.dd HH:mm:ss", Locale.getDefault())
    text = simpleDateFormat.format(date)
```
이런식으로 확장함수로 만들 수도 있다잉 ^_^